### DISCUSSION TITLE
[Question]: v .14 and ffprobe

### DISCUSSION THREAD

**Discussion by RexWheeler** [blakeblackshear/frigate#12529] [Ask A Question] [answered]
### What is your question:

Is there a way in .14 to perform an ffprobe from the UI? This was useful in .13 but I don't see it in .14

**Comment by NickM-27** [comment, marked as answer]
Currently no this is not supported

**Reply by RexWheeler** [reply]
Okay thanks. Please consider adding this to the backlog if it isn't already there.
